ksql-common/src/main/java/io/confluent/ksql/config/ConfigResolver.java

```diff
@@ -49,4 +49,8 @@
    * @param strict if resolution should use strict mode or not.
    */
   Optional<ConfigItem> resolve(String propertyName, boolean strict);
+
+  Optional<ConfigItem> resolveKsqlConfig(String propertyName);
```

Comment:
why did these two methods need exposing?  the idea of the resolver was that it would resolve all valid config, regardless of type.

As far as I can tell `resolveKsqlConfig` is not even called anywhere.

For `resolveStreamsConfig`, what issues were you seeing if you just used `resolve`?